schema/resolvers.js

```diff
@@ -115,7 +115,16 @@ const Resolvers = {
 
       return answer;
     },
-    updateAnswer: (_, args, ctx) => ctx.repositories.Answer.update(args.input),
+    updateAnswer: (_, args, ctx) => {
+      if (args.input.id.endsWith("to")) {
+        return ctx.repositories.Answer.update(
+          mapKeys(args.input, (value, key) => {
```

Comment:
Again not sure we need to map all the keys `assign(args.input, { secondaryLabel: args.input.label })`?